packages/App/src/store/reducers/actions/walletActionsSaga.tsx

```diff
@@ -0,0 +1,54 @@
+import {createAction} from '@reduxjs/toolkit';
+import {WalletInfo} from '../../../types/types';
+
```

Comment:
No need to create separate file for actions dispatched from sagas. The request action does not make sense without success and failure actions, so let's move those to `packages/App/src/store/actions/walletActions.tsx`